### PR TITLE
add more load base split metrics

### PR DESCRIPTION
**Pull request by lhy1024**
Signed-off-by: lhy1024 <admin@liudos.us>

### What problem does this PR solve?

Issue Number: close #xxx 

Problem Summary:
add more load base split metrics

### What is changed and how it works?

Proposal: [xxx](url) 

What's Changed:

### Related changes

- Need to cherry-pick to the release branch

### Check List 

Tests 

- Unit test

### Release note 

```release-note
None.
```

### CODE REVIEW

File: components/raftstore/src/store/worker/split_controller.rs [hunk 1]
```diff
@@ -257,15 +257,24 @@ impl Recorder {
...
+                LOAD_BASE_SPLIT_EVENT
+                    .with_label_values(&["no_enough_key"])
+                    .inc();
                 continue;
             }
             let diff = (sample.left - sample.right) as f64;
             let balance_score = diff.abs() / (sample.left + sample.right) as f64;
             if balance_score >= split_balance_score {
+                LOAD_BASE_SPLIT_EVENT
+                    .with_label_values(&["no_balance_key"])
```
Comment (nolouch) [inline components/raftstore/src/store/worker/split_controller.rs:269]: can we add a histogram for the `balance_score` and  `contained_score`?

Comment (lhy1024) [reply components/raftstore/src/store/worker/split_controller.rs:269]: good idea

**Review by disksing — APPROVED** [review]

**Review by JmPotato — APPROVED** [review]

**Review by nolouch — APPROVED** [review]
lgtm

**Pull request merged**